src/iidm/converter/Parameter.cpp

```diff
@@ -51,6 +55,7 @@ const std::string& Parameter::getStringDefaultValue() const {
 
 const std::vector<std::string>& Parameter::getStringListDefaultValue() const {
     static std::vector<std::string> values;
+    values.clear();
```

Comment:
I think this is not the good way to fix the issue: I think we should return a `vector<std::string>` instead of a reference. The `ConversionParameter::readStringListParameter` already returns a copy, maybe we can rely on the move constructor if we want to avoid two copies.

A mutex would not solve the issue, as the reference can be keep by the caller, and the value modified later.

Reply:
Done